contrib/curl/lib/select.h

```diff
@@ -36,7 +36,8 @@
 
 #if !defined(HAVE_STRUCT_POLLFD) && \
     !defined(HAVE_SYS_POLL_H) && \
-    !defined(HAVE_POLL_H)
+    !defined(HAVE_POLL_H) && \
+    !defined(POLLIN)
```

Comment:
One thing I'm definitely a little leery about is modifying the contributing libraries.

Reply:
That change is from upstream. See this commit https://github.com/curl/curl/commit/7e21e5fd0c709d79ab27c65fd0afff4cf34e0da9

Reply:
I think the correct course of action would be to pull in the most recent release of curl then, rather than modifying this ourselves. Thoughts @starkos?

Reply:
> I think the correct course of action would be to pull in the most recent release of curl

Yes, agree 100%.